app/models/concerns/state.rb

```diff
@@ -15,4 +15,9 @@ module State
   CLOSED_STATE = 'closed'
   SELECTABLE_STATES = [OPEN_STATE, CLOSED_STATE, REJECTED_STATE, HIDDEN_STATE]
   SEARCHABLE_STATES = [OPEN_STATE, CLOSED_STATE, REJECTED_STATE]
+  TODO_LIST_STATES = [SPONSORED_STATE, VALIDATED_STATE, PENDING_STATE]
+  COLLECTING_SPONSORS_STATES = [VALIDATED_STATE, PENDING_STATE]
+  COLLECTING_SPONSORS = 'collecting sponsors'
```

Comment:
@h-lame & @otlaitil now's ready, sorry for false start and thanks for the comments!